src/Containers/Header/Header.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import {
+    OrderStatus,
+    OrderStatusIdentifier,
+} from '@Containers/OrderStatus/OrderStatus';
+import { OrderType, OrderTypeIdentifier } from '@Text/OrderType/OrderType';
+import {
+    OrderPaymentMethod,
+    OrderPaymentMethodTypes,
+} from '@Text/OrderPaymentMethod/OrderPaymentMethod';
+import { Paragraph } from '@Text/Paragraph';
+import Header from './Header';
+import { createStoryTitle } from '../../Constants';
+
+export default {
+    title: createStoryTitle('Order Header'),
+    component: Header,
+    args: {
+        topLeft: (
+            <Paragraph size="1.4rem" bold>
```

Comment:
Suggestion: if you were to make this an OrderID component you can use the slice() logic inside the component instead of repeating it in your application. I would do this for maintainability sake

Reply:
Would having a styled component with these values be a suitable alternative? Where then people can just pass in style overrides through ...props, cause I'm not sure how slice() logic would look like in this case.

Reply:
Styled Component couldn't contain the slice logic defeating the whole point of letting OrderID Component own the logic.

Reply:
Image across Terminal and your ordering service repeating the .slice() logic versus just importing an OrderID component that does it once instead